server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -294,7 +293,7 @@ public StorageGroupProcessor(String systemDir, String storageGroupName,
   }
 
   private void recover() throws StorageGroupProcessorException {
-    logger.info("recover Storage Group  {}", storageGroupName);
+    logger.info("recover Storage Group  {}", virtualstorageGroupName);
```

Comment:
`virtualstorageGroupName` (which I believe should be `virtualstorageGroupId`) is just a number, so the logs of different storage groups may not be distinguishable.

Reply:
fixed~